performance/font-loading.md

```diff
@@ -0,0 +1,64 @@
+# Font Loading
+
+## When to use webfonts
+
+Webfonts allow us to bring emotion, personality and tone to our sites - which can make them more successful.
+
+However, the task of choosing a webfont and implementing it on your site should be approached cautiously. Webfonts have the potential to impact aspects of your sites such as accessibility and page speed. For example:
+
+- Using webfonts may reduce the readability of the site, for example for people with dyslexia, or low vision.
+- Webfonts can negatively impact page performance (additional http requests and increased page weight) especially in markets of strategic importance for Springer Nature like China. This will in turn impact usage numbers.
+
+Our recommendation is that you use a [system font](https://css-tricks.com/snippets/css/system-font-stack/) instead of a webfont. System fonts are extensively tested, incur no performance penalty, and are produced with legibility in mind.
```

Comment:
`**We recommend to use a [system font](https://css-tricks.com/snippets/css/system-font-stack/) instead of a webfont**. System fonts are extensively tested, and incur no performance penalty.`

Highlighting the recommendation.

Also taking into account @foxintherain's feedback about Garamond and legibility, we can't say that "system fonts are legible", per se.

Reply:
Garamond isn't a system font though

Reply:
This might be a pertinent point, knew I should've stuck with Webdings 😂

Reply:
It's probably true though that not all system fonts could be said to have good legibility. So safest to take that bit out I think.

Reply:
My jokes aside about Garamond - I think the main system fonts for OS' (Windows, Apple, Google) absolutely have been tested and very legible - so I think you can leave it in! Maybe I'm getting confused here between browser safe and system os

Reply:
I dunno, I can't verify that all system fonts meet that sweeping statement of being "produced with legibility in mind" so it does feel safer just to remove. Think it's better without.